packages/components/src/PrimaryToolbar/PrimaryToolbar.js

```diff
@@ -53,11 +54,17 @@ class PrimaryToolbar extends Component {
             >
                 <ToolbarContent>
                     {
-                        (bulkSelect || filterConfig || dedicatedAction) &&
+                        (expandAll || bulkSelect || filterConfig || dedicatedAction) &&
                         <ToolbarGroup
                             className="ins-c-primary-toolbar__group-filter pf-m-spacer-md pf-m-space-items-lg"
                             variant="filter-group"
                         >
+                            {
+                                expandAll &&
+                                <ToolbarItem>
+                                    {expandAll}
```

Comment:
If we are going to use just the node the code in each application will duplicate. Can you change this to switch between node and shared implementation?
```JSX
{
  React.isValidElement(bulkSelect) ?
    bulkSelect :
    <ExpandAll {...bulkSelect} />
}
```

Where the `ExpandAll` will be a component similiar to example you used.